aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -339,6 +341,8 @@ module aptos_token::token {
             while (i < token.amount) {
                 let cur_property_version = largest_property_version + i + 1;
                 let new_token_id = create_token_id(token_id.token_data_id, cur_property_version);
+                // TODO(question): This can be very expensive if token_properties is large enough. With many tokens,
+                // can this become expensive enough that mutate_token_properties will fail due to hitting the tx gas limit?
```

Comment:
yes, what is the problem if the TXN fails due to hitting gas limit? we have a mutate_one_token to mutate just one token

Reply:
Yes that could be a work around. I see there's been some threads on limiting token properties size though? Is this still the plan?

Reply:
yes, it landed to limit the property key size

Reply:
Nice